wordpress-mobile/WordPress-iOS
# Reader post card: add ability to mark as seen/unseen

Ref #15355 

This adds the ability to mark a post seen/unseen from a Reader post card.

To test:
- Go to the Reader.
- In a browser, view the same Reader stream.
- In the app, tap the menu button on a card.
- On the menu, verify the seen/unseen option matches that in the Reader for the same post.

| ![app_mark_seen](https://user-images.githubusercontent.com/1816888/104663185-b14c2d80-5689-11eb-812f-95009866849a.jpg) | <img width="827" alt="web_mark_seen" src="https://user-images.githubusercontent.com/1816888/104663197-b7420e80-5689-11eb-806f-2282476f1745.png"> |
|--------|-------|

- Select the menu option.
- Redisplay the menu. Verify the option is now the opposite.
- Refresh the stream in the browser. Verify the option is now the opposite.

| ![app_mark_unseen](https://user-images.githubusercontent.com/1816888/104663222-c45efd80-5689-11eb-9b00-e8b1c784d83e.jpg) | <img width="832" alt="web_mark_unseen" src="https://user-images.githubusercontent.com/1816888/104663226-c759ee00-5689-11eb-9ad4-2f5b041dc999.png"> |
|--------|-------|

PR submission checklist:
